pipeline/schemas/pipeline_file.py

```diff
@@ -42,4 +44,7 @@ class PipelineFileDirectUploadFinaliseCreate(BaseModel):
 class PipelineFileGet(BaseModel):
     id: str
     name: str
-    hex_file: FileGet
+    # This is a legacy field that is replaced by 'file'.
+    # Can be deprecated once no clients are using it.
```

Comment:
I suppose we can consider it deprecated immediately, and then _remove_ it once it's no longer used.

Pydantic has no explicit support for deprecated fields from what I can see, but one could use a `Field(None, deprecated=True)` convention if desired. Our versioning is so informal today that it probably doesn't mean much to add this though.